android/src/main/java/org/mozilla/mozstumbler/client/mapview/MapFragment.java

```diff
@@ -90,6 +90,7 @@
     private LowResMapOverlay mLowResMapOverlayLowZoom;
     private Overlay mCoverageTilesOverlayLowZoom;
     private Overlay mCoverageTilesOverlayHighZoom;
+    private Overlay mCoverageTilesOverlayDefaultZoom;
```

Comment:
Hmm, not sure what to call this, but saying 'default' here is confusing. Unfortunately, "mCoverageTilesOverlayUnlimitedModeAtLowZoom" is the only thing I can think of :) that makes the purpose of this totally clear.